common/changes/@itwin/analytical-backend/pankhur94-add-preview-support_2025-04-17-20-27.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/analytical-backend",
+      "comment": "added tsdoc.config file",
```

Comment:
these changelogs are user facing, and this change is internal only. id remove the comment on all new changefiles you created in this pr